### ISSUE TITLE
Running UBI images gives openssl command not found

### ISSUE THREAD

**Issue by aguibert** [OpenLiberty/ci.docker#130]
Example output:
```
$ docker run openliberty/open-liberty:full-java8-openj9-ubi
/bin/sh: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
/opt/ol/helpers/runtime/docker-server.sh: line 21: openssl: command not found
/bin/sh: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)

Launching defaultServer (Open Liberty 19.0.0.12/wlp-1.0.35.cl191220191120-0300) on Eclipse OpenJ9 VM, version 1.8.0_232-b09 (en_US)
[AUDIT   ] CWWKE0001I: The server defaultServer has been launched.
[AUDIT   ] CWWKG0093A: Processing configuration drop-ins resource: /opt/ol/wlp/usr/servers/defaultServer/configDropins/defaults/keystore.xml
[AUDIT   ] CWWKG0093A: Processing configuration drop-ins resource: /opt/ol/wlp/usr/servers/defaultServer/configDropins/defaults/open-default-port.xml
```

**Comment by aguibert** [comment]
Side note: The `setlocale` errors appear to be cause by the JDK image we are building on top of:

```
$ docker run adoptopenjdk/openjdk8-openj9:ubi
/bin/bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
```

**Comment by aguibert** [comment]
Raised this issue for the locale warnings: https://github.com/AdoptOpenJDK/openjdk-docker/issues/255

**Comment by arthurdm** [comment]
Thanks @aguibert - I have opened this issue for the missing `openssl` package in the AdoptOpenJDK board: https://github.com/AdoptOpenJDK/openjdk-docker/issues/256

**Comment by mnesarco** [comment]
Hi Friends, I know this is marked as closed, but i still get the problem with openssl:

FROM openliberty/open-liberty:19.0.0.12-full-java11-openj9-ubi

```
/bin/sh: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
/opt/ol/helpers/runtime/docker-server.sh: line 21: openssl: command not found

```

I do not understand  how to solve this.

**Comment by mnesarco** [comment]
My current workaround is:

```
FROM openliberty/open-liberty:19.0.0.12-kernel-java11-openj9-ubi
USER root
RUN yum install -y glibc-langpack-en openssl

```

**Comment by arthurdm** [comment]
thanks for the feedback @mnesarco - we are in the process of updating the builds to pickup the latest commits, so that we get Andy's fix (#131), which will remove the dependency on openssl.
